pubspec.yaml

```diff
@@ -4,13 +4,15 @@ version: 0.0.1
 homepage: https://github.com/flutter/devtools
 
 environment:
-  sdk: '>=2.0.0-dev <3.0.0'
+  sdk: '>=2.1.0 <3.0.0'
```

Comment:
FWIW - this constraint prevents the SDK included in Flutter v1.0 from being able to activate this package (it shipped with `2.1.0-dev.9.4.flutter-f9ebf21297` and a `-dev` suffix makes that "less than" 2.1.0).

I suspect we're not concerned but since I noticed it when adding DC integration tests and it's a little counter-intuitive (we consider Flutter v1.0 shipped with Dart 2.1, but semver doesn't see it that way!) I thought it worth noting.

Reply:
Since we're effectively just using this constraint in the pub global activated server app, I think we can change the lower bound to Flutter's.